app/modules/transactions/components/cancel_transaction.svelte

```diff
@@ -0,0 +1,64 @@
+<script>
+  import Spinner from '#components/spinner.svelte'
+  import Flash from '#lib/components/flash.svelte'
+  import { transactionIsCancellable, updateTransactionState } from '#transactions/lib/transactions_actions'
+  import { i18n } from '#user/lib/i18n'
+
+  export let transaction
+
+  let cancelling, flash
+  async function askConfirmationToCancel () {
+    app.execute('ask:confirmation', {
+      confirmationText: i18n('transaction_cancel_confirmation'),
+      action: cancel,
+    })
+  }
+
+  async function cancel () {
+    try {
+      cancelling = true
+      transaction = await updateTransactionState({ transaction, state: 'cancelled' })
```

Comment:
When i click on "Cancel", same next actions are still visible. If i refresh it all good.

![2024-02-22_08-00](https://github.com/inventaire/inventaire-client/assets/5363918/2b46fd54-6ef2-445f-9d16-87aa228688c7)

Reply:
As above, that should now be fixed by https://github.com/inventaire/inventaire-client/commit/99fb8a23774fc671457043d2434b20b96e907c0a and https://github.com/inventaire/inventaire/pull/733